testing/qt5-qtserialport/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com> 
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=qt5-qtserialport
+pkgver=5.6.0
+pkgrel=1
+pkgdesc="Interface to serial ports for Qt application"
+url="http://qt-project.org/wiki/QtSerialPort"
+arch="all"
+license="LGPL"
+depends="qt5-qtbase"
+depends_dev="$depends"
+makedepends="$depends_dev qt5-qtbase-dev"
+install=""
+subpackages="$pkgname-dev"
+
+_ver=${pkgver/_/-}
+_ver=${_ver/beta0/beta}
+_ver=${_ver/rc0/rc}
+_V=${_ver/rc/RC}
```

Comment:
Nope, this is not from [unmaintained/qtserialport/APKBUILD](https://github.com/alpinelinux/aports/blob/master/unmaintained/qtserialport/APKBUILD).

Well, if you haven’t written it from scratch, then you should rather update unmaintained/qtserialport and then move it into testing (or vice versa), instead of adding it as a new aport.